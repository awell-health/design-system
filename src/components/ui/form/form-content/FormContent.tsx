import { cn } from '../../../../lib/utils';

interface Props extends React.InputHTMLAttributes<HTMLDivElement> {
  children: React.ReactNode;
  className?: string;
}

const FormContent = (props: Props) => {
  const { children, className } = props;
  return (
    <div className={cn('flex flex-col border border-gray-200 rounded-lg p-4 ', className)}>
      {children}
    </div>
  );
};

export default FormContent;
