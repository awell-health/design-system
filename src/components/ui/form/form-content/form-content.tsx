import { cn } from '../../../../lib/utils';

interface Props extends React.InputHTMLAttributes<HTMLDivElement> {
  children: React.ReactNode;
  className?: string;
}

export const FormContent = (props: Props) => {
  const { children, className } = props;
  return (
    <div className={cn('flex flex-col border border-gray-200 rounded-lg p-4 ', className)}>
      {children}
    </div>
  );
};
