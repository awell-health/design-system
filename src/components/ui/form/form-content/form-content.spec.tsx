import React from 'react';
import { render } from '@testing-library/react';
import { expect, it, describe } from 'vitest';
import { FormContent } from './form-content';

describe('FormContent', () => {
  it('matches snapshot', () => {
    const { container } = render(
      <FormContent>
        <div>Child test content</div>
        <div>Child test content</div>
        <div>Child test content</div>
      </FormContent>
    );
    expect(container).toMatchSnapshot();
  });
});
