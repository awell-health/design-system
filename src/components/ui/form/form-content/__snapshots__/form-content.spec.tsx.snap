// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FormContent > matches snapshot 1`] = `
<div>
  <div
    class="flex flex-col border border-gray-200 rounded-lg p-4"
  >
    <div>
      Child test content
    </div>
    <div>
      Child test content
    </div>
    <div>
      Child test content
    </div>
  </div>
</div>
`;
